### PR TITLE
add the props to have the small actions in the ActionBar component

### PR DESCRIPTION
**Pull request by pochretien**
image

much better without spaces for review https://github.com/coveo/react-vapor/pull/535/files?w=1

### CODE REVIEW

File: src/components/actions/ActionBar.tsx [hunk 1]
```diff
@@ -24,6 +24,7 @@ export interface IActionBarOwnProps extends React.ClassAttributes<ActionBar> {
   onClearItemFilter?: () => void;
   extraContainerClasses?: string[];
   removeDefaultContainerClasses?: boolean;
+  withsSmallActions?: boolean;
```
Comment (MichelML) [inline src/components/actions/ActionBar.tsx]: withsSmallActions --> withSmallActions

File: src/components/actions/examples/ActionBarExamples.tsx [hunk 2]
```diff
@@ -64,6 +64,10 @@ export class ActionBarExamples extends React.Component<any, any> {
           <label className='form-control-label'>Action bar with default classes turned off and extra classes</label>
           <ActionBar actions={actions} removeDefaultContainerClasses extraContainerClasses={['coveo-table-actions-container', 'p2']} />
         </div>
+        <div className='form-group'>
+          <label className='form-control-label'>Action bar with small actions</label>
+          <ActionBar actions={actions} withsSmallActions={true} />
```
Comment (MichelML) [inline src/components/actions/examples/ActionBarExamples.tsx]: i don't know if it is in the style guide, but we usually omit the ={true} for boolean prop, you can only put `withSmallActions` and it does the same

File: src/components/actions/ActionBar.tsx [hunk 3]
```diff
@@ -53,6 +54,7 @@ export interface IActionBarProps extends
 export class ActionBar extends React.Component<IActionBarProps, any> {
   static defaultProps: Partial<IActionBarOwnProps> = {
     extraContainerClasses: [],
+    withsSmallActions: false,
```
Comment (sroy3) [inline src/components/actions/ActionBar.tsx]: You don't have to define it. `true` or `undefined` is good enough.

Comment (pochretien) [reply src/components/actions/ActionBar.tsx]: I think its better to handle a boolean always defined instant of an undefined value. And more, we can see faster the default state for an actionBar rendered

**Pull request merged**